ui/desktop/electron-app/src/services/client-daemon-manager.js

```diff
@@ -154,6 +156,8 @@ const searchCliCommand = (requestData) => {
   }
   const sanitizedToken = sanitizer.base62EscapeAndValidate(requestData.token);
 
+  log.info('logging search command:', searchCommand);
```

Comment:
I think, before we log `searchCommand` we should delete the `token` field from it, we can not log sensitive data such as tokens.

Reply:
Taking as example a line of the generated log:
```
{
  method: 'GET',
  path: 'http://internal.boundary.local/v1/search?force_refresh=true&query=&auth_token_id=at_JhnjID5a06&resource=targets',
  socketPath: '/Users/ccorvo/.boundary/socket/daemon.sock'
}
```

Inside the path, I see the `auth_token_id` is this something safe to log?